### DISCUSSION TITLE
lotus-miner start fail (checksum mismatch in param file)

### DISCUSSION THREAD

**Discussion by ahffkTid** [filecoin-project/lotus#7782] [Storage Provider]
### Checklist

- [ ] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [ ] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [ ] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [ ] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [ ] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [ ] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [ ] I did not make any code changes to lotus.

### Lotus component

lotus miner - mining and block production

### Lotus Version

```text
lotus version 1.10.1+mainnet+git.d920b47e7
```

### Describe the Bug

when i start the lotus-miner this file keeps getting deleted
(v28-stacked-proof-of-replication-merkletree-poseidon_hasher-8-8-0-sha256_hasher82a357d2f2ca81dc61bb45f4a762807aedee1b0a53fd6c4e77b46a01bfef7820.params)
even this file Copy or Move  or Download for params directory  same issue repeats
when i start lotus-miner a miner-log like the in the picture appearsthe param file(45GB) is deleted
![miner_error](https://user-images.githubusercontent.com/88740654/128922475-4ad3f119-6b83-4210-bcaa-5640b9b6df91.png)

**Comment by Reiers** [comment]
**"checksum mismatch"**
This can indicate a hw problem.

Try to redownload the params to another disk, and start the machine with just 1 RAM stick.
If that works, then you have a bad RAM stick _or a faulty disk._ 

I sent this to discussion for troubleshooting. Because I was not able to reproduce this issue, but I have seen it in the past and every time it was a hardware issue. 

For re-download:
32GiB Params
`lotus-miner fetch-params 32GiB
`
64GiB Params
`lotus-miner fetch-params 64GiB`
